TheMET/TheMET/ViewControllers/CulturesSectionViewController.swift

```diff
@@ -0,0 +1,204 @@
+//
+//  CulturesSectionViewController.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 12/12/2023.
+//
+
+import Foundation
+import UIKit
+import MetAPI
+
+class CulturesSectionViewController: UIViewController {
+    
+    private let culturesService = CulturesService.standart
+    
+    private let metAPI = MetAPI()
+    
+    private let culturesLabel: UILabel = UILabel()
+    
+    private var loadingCulturesNames: [String] = []
+    
+    private let loadingCulturesCatalogView = CatalogSectionLoadingView.constractView(configuration: .catalogSectionLoading)
+    private let failedCulturesCatalogView = CatalogSectionTapToReloadView.constractView(configuration: .catalogSectionTapToReload)
+    private let loadedCulturesCatalogView = CulturesSectionContentView.constructView()
+    
+    private var loadedCultures: [CatalogSectionCellData<String>] = []
+    
+    var contentStatus:LoadingStatus<[CatalogSectionCellData<String>]> = .loading {
+        didSet {
+            self.updateContent()
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.culturesLabel.translatesAutoresizingMaskIntoConstraints = false
+        self.view.addSubview(culturesLabel)
+        self.culturesLabel.apply(font: NSLocalizedString("serif_font", comment: ""), color: UIColor(named: "pear"), fontSize: 16, title: NSLocalizedString("catalog_screen_cultures_section_title", comment: ""))
+        self.culturesLabel.textAlignment = .center
+        NSLayoutConstraint.activate([
+            self.culturesLabel.leadingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leadingAnchor, constant: 20),
+            self.culturesLabel.trailingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.trailingAnchor, constant: -20),
+            self.culturesLabel.topAnchor.constraint(equalTo: self.view.topAnchor),
+            self.culturesLabel.heightAnchor.constraint(equalToConstant: 60)
+            ])
+        self.add(catalogSubview: self.failedCulturesCatalogView)
+        self.add(catalogSubview: self.loadedCulturesCatalogView)
+        self.add(catalogSubview: self.loadingCulturesCatalogView)
+        self.failedCulturesCatalogView.onButtonTap = { [weak self] in
+            self?.reloadButtonDidTap()
+        }
+        self.loadedCulturesCatalogView.onCultureCellTap = { [weak self] culture in
+            self?.cultureCellDidTap(culture)
+        }
+        self.loadedCulturesCatalogView.onCultureCellWillDisplay = { [weak self] culture in
+            self?.loadParticularCultureCellDataIfNeeded(culture: culture)
+        }
+        self.reloadCatalog(culturesList: self.culturesService.culturesList)
```

Comment:
`self.culturesService.culturesList` is a published property, which means its better to observe it and react to it changes rather than just reload it once.

Reply:
Done

Reply:
You don't need to `reloadCatalog` at line 65 anymore cause you've already did it by subscribing to `culturesList` changes on line 39